package.json

```diff
@@ -35,7 +35,7 @@
     "calculate-cache-key-for-tree": "^2.0.0",
     "caniuse-api": "^3.0.0",
     "ember-cli-babel": "^7.19.0",
-    "ember-cli-typescript": "^3.1.3",
+    "ember-cli-typescript": "^4.0.0",
```

Comment:
This should be split out, I think. It seems unrelated to the overall goal of testing Embroider.

Reply:
But updating the dependency addressed an issue where Babel detected duplicate plugins.